spec.json

```diff
@@ -10,11 +10,16 @@
                 "The motor class provides a uniform interface for using motors with",
                 "positional and directional feedback such as the EV3 and NXT motors.",
                 "This feedback allows for precise control of the motors. This is the",
-                "most common type of motor, so we just call it `motor`."
+                "most common type of motor, so we just call it `motor`.",
+                "",
+                "The way to configure a motor is to set the '_sp' attributes when",
+                "calling a command or before. Only in 'run_direct' mode attribute",
+                "changes are processed immediately, in the other modes they only",
+                "take place when a new command is issued."
                 ],
             "docsLink": "http://www.ev3dev.org/docs/drivers/tacho-motor-class/",
             "systemClassName":  "tacho-motor",
-            "systemDeviceNameConvention":  "motor{0}",
+            "systemDeviceNameConvention":  "*",
```

Comment:
It's a workaround for the fact that we have two types of motors - the Firgellis have a device name of `linear*` while all others are `motor*` - if you instantiate using `ev3dev.Motor` and then change the driver type to a linear motor, the device name changes to `linear*`.

You can always instantiate the right motor type and get the more precise motor name